extensions/braze/braze-currents-users-event.example.1.json

```diff
@@ -0,0 +1,25 @@
+{
+  "braze:id": "a1234567-89ab-cdef-0123-456789abcdef",
+  "braze:time": 1477502783,
```

Comment:
- Where is "braze:time" defined in the schema? I didn't see it in the `braze-currents-users-event.schema.json`
- What format are you expecting for the time? Do you need AEP to validate its value and accept/reject if the value is invalid?

Reply:
It was in the schema previously, until someone pointed out that we could just use the normal XDM `timestamp` field. Should we omit both it and the `timestamp` from the example payloads?

Our timestamp is just a Unix timestamp, in seconds.

Reply:
Let's remove these two from all examples, and add an example for "timestamp" field. 
Note that the "timestamp" is defined as a date-time string, which means it needs to conform to ISO-8601 standard. 
Refer: https://experienceleague.adobe.com/docs/experience-platform/xdm/schema/field-constraints.html?lang=en#basic-types